Assets/Scripts/AnimalMovement.cs

```diff
@@ -0,0 +1,42 @@
+using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+namespace AnimalPOV
+{
+    public class NewBehaviourScript : MonoBehaviour
+    {
+        public interface IControl
+        {
+
+        }
+       public class Rat : IControl
```

Comment:
This is not Rat class, this is MovementParameters

Reply:
Also, please add [Serializable] attribute to this class. It will allow as to serialize this class in unity later